_posts/2020-05-01-newsletter.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Origami Newsletter, April 2020
+description: This issue features the Origami Component Survey, Automating Visual Testing of Origami Components and moving from CircleCI to GitHub Actions
+author: Jake Champion
+tags:
+  - Newsletter
+---
+
+<abbr title="Too long; didn't read">
+	<strong>
+	TL;DR:
+	</strong>
+</abbr> {{page.description}}
+
+## Top Things
+
+These are some of the bigger things we've done over the last month.
+
+### Origami Components Survey
+
+This quarter we are aiming to find out what our users know about Origami, how much of Origami they use and how we can improve our products for all of our users.
```

Comment:
I think maybe you're thinking of the upcoming specialist survey here? The survey that went out was much more focused on understanding the barriers that teams have to developing their own Origami (or Origami-compatible) components

Reply:
ah yeah, sorry somehow I misinterpreted it